resources/sdk/purecloudgo/templates/apiresponse.mustache

```diff
@@ -48,7 +55,7 @@ type APIError struct {
 	MessageParams     map[string]interface{} `json:"messageParams,omitempty"`
 	Code              string                 `json:"code,omitempty"`
 	ContextID         string                 `json:"contextId,omitempty"`
-	Details           []string               `json:"details,omitempty"`
+	Details           []APIErrorDetail       `json:"details,omitempty"`
```

Comment:
The APIErrorDetail struct is defined as `Detail` in https://github.com/MyPureCloud/platform-client-sdk-go/blob/master/platformclientv2/detail.go.
You could update this to make use of that struct, either way works.

Reply:
Ah I missed that. I'll switch to the existing struct, thanks.